metadata-load/src/test/java/uk/ac/ebi/ampt2d/metadata/parser/AnalysisFileTypeFromSetTest.java

```diff
@@ -0,0 +1,95 @@
+/*
+ *
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.metadata.parser;
+
+import org.apache.xmlbeans.XmlException;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+import uk.ac.ebi.ena.sra.xml.AnalysisFileType;
+import uk.ac.ebi.ena.sra.xml.AnalysisSetType;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@RunWith(MockitoJUnitRunner.class)
+public class AnalysisFileTypeFromSetTest {
+    @Test
+    public void testAnalysisFileParser() throws Exception {
+        String xmlStr = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n" +
```

Comment:
we can read xml file from resources , if we want .

Reply:
+1

Reply:
Done